.github/ISSUE_TEMPLATE/install-bug-report.yml

```diff
@@ -0,0 +1,88 @@
+name: 🐛 Install Bug Report
+description: Resport an Egg install issue
+title: "[Install Bug]: <Egg Name>"
+labels: ["install bug", "not confirmed"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you see any of the following, go to [Discord](https://discord.gg/pterodactyl) and report your error in a support channel.
+        * `A fatal error was encountered while starting this server.`
+        * `No server egg configuration could be located; aborting startup.`
+
```

Comment:
We should add the warning you get if you try to import a v2 egg in to a panel version older then v1.8 to come to the discord